.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - "$HOME/.composer/cache"
+
+env:
+  - COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
```

Comment:
you cannot have both. The idea is to make separate builds for each. So one where you prefer the latest (stable) and one where you prefer the lowest.

So you should have a matrix of builds, like here https://github.com/hostnet/form-handler-bundle/blob/master/.travis.yml#L15-L30